.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,8 @@
-### Description
-**Outline the issue here:**
+
 
 ---
 
 ### Environment summary
 
-**Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, edge build)  
-Answer here:
-
-**CLI Version:** What version of the CLI and modules are installed? (Use `az --version`)  
-Answer here:
-
-**OS Version:** What OS and version are you using?  
-Answer here:
-
-**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, Bash on Windows)  
-Answer here:
+Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows)  
```

Comment:
Add a pair triple backtick after this to ensure whatever user paste here is well-formatted.